test/rfqt_test.ts

```diff
@@ -488,34 +361,22 @@ describe(SUITE_NAME, () => {
         });
 
         it('should not return order if maker allowances are not set', async () => {
-            const sellAmount = new BigNumber(100000000000000000);
-
             const wethContract = new WETH9Contract(contractAddresses.etherToken, provider);
             await wethContract
                 .approve(contractAddresses.erc20Proxy, new BigNumber(0))
                 .sendTransactionAsync({ from: takerAddress });
 
-            const mockedApiParams = {
-                sellToken: contractAddresses.etherToken,
-                buyToken: contractAddresses.zrxToken,
-                sellAmount: sellAmount.toString(),
-                buyAmount: undefined,
-                takerAddress,
-            };
             return rfqtMocker.withMockedRfqtFirmQuotes(
                 [
                     {
-                        endpoint: 'https://mock-rfqt1.club',
+                        ...DEFAULT_RFQT_RESPONSE_DATA,
                         responseData: ganacheZrxWethOrder1,
-                        responseCode: 200,
-                        requestApiKey: 'koolApiKey1',
-                        requestParams: mockedApiParams,
                     },
                 ],
                 async () => {
                     const appResponse = await request(app)
                         .get(
-                            `${SWAP_PATH}/quote?buyToken=ZRX&sellToken=WETH&sellAmount=${sellAmount.toString()}&takerAddress=${takerAddress}&intentOnFilling=true&excludedSources=Uniswap,Eth2Dai,Kyber,LiquidityProvider&skipValidation=true`,
+                            `${SWAP_PATH}/quote?buyToken=ZRX&sellToken=WETH&sellAmount=${DEFAULT_RFQT_RESPONSE_DATA.toString()}&takerAddress=${takerAddress}&intentOnFilling=true&excludedSources=${DEFAULT_EXCLUDED_SOURCES}&skipValidation=true`,
```

Comment:
`sellAmount=${DEFAULT_RFQT_RESPONSE_DATA.toString()}` should be `sellAmount=${DEFAULT_SELL_AMOUNT}`

Reply:
🤦 good catch, thanks